src/docs/installation.md

```diff
@@ -76,6 +78,57 @@ $ cd <projectName>
 $ npm start
 ```
 
+*If you got some issues like:*
```

Comment:
Since this seems to be an issue that affects relatively few users, this could best be mentioned in a GitHub issue